lms/services/lti_outcomes.py

```diff
@@ -159,14 +95,39 @@ def record_result(  # pylint:disable=no-self-use
             If the submission date matches an existing submission then the
             existing submission is updated rather than creating a new submission.
         """
-        body = REPLACE_RESULT_REQUEST_TEMPLATE.render(
-            lis_result_sourcedid=outcomes_request_params.lis_result_sourcedid,
-            score=score,
-            lti_launch_url=lti_launch_url,
-            submitted_at=submitted_at,
+
+        request = {
+            "resultRecord": {
+                "sourcedGUID": {
+                    "sourcedId": outcomes_request_params.lis_result_sourcedid
+                }
+            }
+        }
```

Comment:
A lot cleaner than the previous `REPLACE_RESULT_REQUEST_TEMPLATE`, inlined where it's used, and contains only the unconditional parts on their own :+1: